ci-scripts/yocto2.groovy

```diff
@@ -131,10 +131,22 @@ void runYoctoCheckLayer(String yoctoDir) {
 
 
 void archiveCache(String yoctoDir, boolean doArchiveCache, String yoctoCacheArchivePath) {
-    if (doArchiveCache && yoctoCacheArchivePath?.trim()) {
-        stage("Archive cache") {
-            sh "rsync -trpgO ${yoctoDir}/build/downloads/ ${yoctoCacheArchivePath}/downloads/"
-            sh "rsync -trpgO ${yoctoDir}/build/sstate-cache/ ${yoctoCacheArchivePath}/sstate-cache"
+    def sC = sh script:"test -d ${yoctoCacheArchivePath}/sstate-cache && test -d ${yoctoCacheArchivePath}/downloads", returnStatus:true
+    if (sC != 0) {
+        echo "Cache dirs arn't mounted"
+    } else {
+        echo "Cache dirs are mounted"
+        if (doArchiveCache && yoctoCacheArchivePath?.trim()) {
+	    
+            stage("Archive cache") {
+	            try {
```

Comment:
Indentation.

Reply:
Which exactly?

Reply:
4 spaces used for indentation everywhere else, this is 8 (or a tab perhaps?).